compiler-rt/test/sanitizer_common/TestCases/Linux/pthread_join.cpp

```diff
@@ -4,10 +4,15 @@
 // FIXME: Crashes on some bots in pthread_exit.
 // RUN: %run %t %if tsan %{ 0 %} %else %{ 1 %}
 
+// Check interceptors' return codes are the same without sanitizers.
```

Comment:
I don't see value in this test
-fno-sanitize=all removes runtime, we don't have interceptors.

Reply:
I meant it as a sanity check to double check the interceptors returns the same error code as the intercepted functions.
Not strictly required so happy to remove.